cloudinventario

```diff
@@ -101,9 +169,17 @@ def main(args):
   config = loadConfig(args.config)
   cinv = CloudInventario(config)
 
+  # load prometheus (create registry, define metrics)
+  METRICS, REGISTRY = loadPrometheus()
+  PUSHADD_PROMETHEUS = lambda registry: pushadd_to_gateway(
+            (os.getenv("PROMETHEUS_GATEWAY") or config['prometheus']['gateway']),
+            (os.getenv("PROMETHEUS_JOB") or config['prometheus']['job']),
```

Comment:
Script will probably crash, if one of gateway and job is not defined, but only `gateway` is required, and prometheus should init only if its present.
`job` is optional with default ie. `Cloudinventario`

Reply:
Add default value for JOB parameter in commit '74c1c26'.